docs/src/transfer-hook-interface/introduction.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Introduction
+---
+
+The Transfer Hook Interface is one of several interfaces introduced within the
+Solana Program Library that can be implemented by any Solana program.
+
+Token-2022 implements this interface, as described in the [Transfer Hook
+Extension Guide](../../token-2022/extensions#transfer-hook). Additionally, a
+[reference
+implementation](https://github.com/solana-labs/solana-program-library/tree/master/token/transfer-hook/example)
+can be found in the SPL GitHub repository, detailing
+how one might implement this interface in their own program.
+
+Transfer Hook is designed to allow programs to "hook" additional functionality
+into token transfers. The program performing the transfer can then CPI into the
+transfer hook program using the interface-defined instructions to perform the
+custom functionality.
+
+In the case of Token-2022, the presence of a transfer hook program is assigned
+to a mint using a mint extension, and this extension tells Token-2022 which
+program to CPI to whenever a transfer is conducted. However, this particular
+implementation is not required, and the interface can be implemented in a
+variety of ways!
```

Comment:
I was suggesting here that you don't need to have the transfer hook extension as a Mint extension specifically - as Token2022 does. You might integrate it some other way. Is this accurate? Or are we saying that `TransferHook` is part of the Token interface and that's how you have compatibility with a Transfer Hook program?

Reply:
Ah I see what you mean. It seems like the term "interface" is overloaded between the transfer hook interface and a user of the interface, and in this case, you mean the second one.

You can say something like "The transfer hook interface doesn't need to be used through Token-2022, and could be used by any program that wishes to add custom functionality to an instruction resembling `Transfer` from spl-token."

But to be honest, I think it's better to remove the sentence to avoid confusion.